src/store/dashboard/awsDashboard/awsDashboardWidgets.ts

```diff
@@ -143,6 +147,7 @@ export const storageWidget: AwsDashboardWidget = {
     computedReportItem: ComputedReportItemType.usage,
     computedReportItemValue: ComputedReportItemValueType.total,
     titleKey: messages.dashboardDailyUsageComparison,
+    chartName: 'dashboardDailyUsageComparisonStorage',
```

Comment:
Would you mind prefixing the chart names with "aws"; for example, "awsStorageChart"